src/elements/demo-image/DemoImageElementForm.tsx

```diff
@@ -60,7 +60,13 @@ export const ImageElementForm: React.FunctionComponent<Props> = ({
       fieldViewSpec={fieldViewSpecs.optionDropdown}
       errors={errors.optionDropdown}
     />
-    <ImageView fieldViewSpec={fieldViewSpecs.mainImage} />
+    <ImageView
+      fieldViewSpec={fieldViewSpecs.mainImage}
+      onChange={(_, __, ___, description) => {
+        fieldViewSpecs.altText.update(description);
+        fieldViewSpecs.caption.update(description);
```

Comment:
Does this alter the way in which errors are handled? As my branch is rebased from this one.

Reply:
I don't think we have validation for the `ImageView` component at the moment – is that what you're referring to here? I may have misunderstood, but I don't think that this should affect your work.